src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,95 @@
+@use "../utils/extends";
+@use "../utils/variables" as *;
+
+body {
+  margin: 0;
+  padding: 0;
+  height: 100vh;
+  background-color: #FFEEDB;
+  @extend %centre-flex;
+  font-size: 44px;
+  font-weight: 500;
+  font-family: "Roboto", sans-serif;
+}
+
+.watch {
+  position: relative;
+  @extend %watch-size;
+  @extend %watch-radius;
+  background-color: #F9DC5C;
+  border: 16px solid #0b4f6c43;
+}
+
+.watch__centre {
+  @extend %centre-flex;
+  @extend %watch-centre;
+  flex-direction: column;
+  text-transform: uppercase;
+  @extend %watch-radius;
+  height: calc(#{$watch-size * 0.095});
+  width: calc(#{$watch-size * 0.095});
+  background-color: #3185FC;
+  z-index: 5;
+  transform: translate(-#{$centre-absolute}, -#{$centre-absolute});
+}
+
+.watch__hand1 {
+  @extend %watch-centre;
+  background-color: #EFBCD5;
+  height: calc(#{$watch-size} * 0.02);
+  width: calc(#{$watch-size} * 0.30);
+  top: calc(50% - #{$watch-size} * 0.02);
+  z-index: 3;
+  left: #{$centre-absolute};
+  @extend %clock-hand-centre;
+  @extend %animaton-time-directon;
+  transform-origin: 0% 50%;
+  animation: 3600s linear -2700s infinite;
```

Comment:
what you actually need here: 
- animation duration
- animation name
- animation iteration count

make sure to fix it for the second one as well

Reply:
add the `move` animation here – you've created it, but don't use it